java-allocation-instrumenter/pom.xml

```diff
@@ -223,4 +223,36 @@
       <organization>Google Inc.</organization>
     </developer>
   </developers>
+  
+  <profiles>
+    <profile>
+      <id>jdk8</id>
+      <activation>
+        <jdk>[1.8,)</jdk>
+      </activation>
+      <!-- Disable doclint under JDK 8 -->
```

Comment:
Can you explain this to me a bit further?  Is there something weird about the code, or about JDK8's doclint?